substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfDebugInfo.java

```diff
@@ -201,4 +203,8 @@ public DwarfARangesSectionImpl getARangesSectionImpl() {
     public DwarfLineSectionImpl getLineSectionImpl() {
         return dwarfLineSection;
     }
+
+    public ELFMachine getELFMachine() {
```

Comment:
You don't need an accessor if you make `elfMachine` a public final field.

Reply:
Sure, no point making the API surface larger than it needs to be. Fixed